internal/store/horizontalpodautoscaler.go

```diff
@@ -56,6 +56,23 @@ var (
 
 func hpaMetricFamilies(allowAnnotationsList, allowLabelsList []string) []generator.FamilyGenerator {
 	return []generator.FamilyGenerator{
+		*generator.NewFamilyGenerator(
+			"kube_horizontalpodautoscaler_info",
+			"Information about this autoscaler.",
+			metric.Gauge,
+			"",
+			wrapHPAFunc(func(a *autoscaling.HorizontalPodAutoscaler) *metric.Family {
+				return &metric.Family{
+					Metrics: []*metric.Metric{
+						{
+							LabelKeys:   []string{"scaletargetref_api_version", "scaletargetref_kind", "scaletargetref_name"},
```

Comment:
We should probably include the HPA's name and namespace in the labels, otherwise we can't associate a target with an HPA resource :)

Reply:
These labels are added by default in `store.wrapHPAFunc` 😄 
https://github.com/kubernetes/kube-state-metrics/blob/f5b42be4455d5ab37063c848d924f28c4e5b0ea1/internal/store/horizontalpodautoscaler.go#L282-L285

Reply:
Ah, ok forgot about that completely :+1: